src/student/student.controller.ts

```diff
@@ -18,16 +18,19 @@ import { UpdateStudentDetailsDto } from './dto/update-student-details.dto';
 import { UpdateStudentDetailsResponse } from '../types/student/update-student-details-response';
 import { CheckUniquePropertiesGuard } from '../guards/check-unique-properties.guard';
 import { ChangeStudentStatusResponse } from '../types/student/change-student-status-response';
+import { JwtAuthGuard } from '../auth/jwt-auth.guard';
 
 @Controller('student')
 export class StudentController {
   constructor(@Inject(StudentService) private studentService: StudentService) {}
 
+  @UseGuards(JwtAuthGuard)
   @Get('/available-students')
   async getAvailableStudents(): Promise<Omit<OneStudentResponse, 'degrees'>[]> {
     return await this.studentService.getStudents(StudentStatus.available);
   }
 
+  @UseGuards(JwtAuthGuard)
```

Comment:
Why do we protect  by guard only these two EPs, but rest not?

Reply:
this is only for tests now we must choose witch endpoints we must protect